Wireframe/css/style.css

```diff
@@ -0,0 +1,229 @@
+/* Stlyles 
```

Comment:
Hey, can you remove code that is commented out unless you have a question about it? Thanks! (It's a bit hard to read code with lots of "dead code" in it.)

Reply:
Sure,  deleted some unecessary coments on my css.